### PR TITLE
Rename projects

### PR DESCRIPTION
**Pull request by eneufeld**

### CODE REVIEW

File: backend/plugins/org.eclipse.emfcloud.coffee.workflow.dsl/src/org/eclipse/emfcloud/coffee/workflow/dsl/Workflow.xtext [hunk 1]
```diff
@@ -8,7 +8,7 @@
  * 
  * SPDX-License-Identifier: EPL-2.0 OR MIT
  ******************************************************************************/
-grammar com.eclipsesource.workflow.dsl.Workflow with org.eclipse.xtext.common.Terminals
+grammar org.eclipse.emfcloud.coffee.workflow.dsl.Workflow with org.eclipse.xtext.common.Terminals
 
 import "http://www.eclipse.org/emf/2002/Ecore" as ecore
 generate workflow "http://www.eclipsesource.com/workflow/dsl/Workflow"
```
Comment (ndoschek) [inline backend/plugins/org.eclipse.emfcloud.coffee.workflow.dsl/src/org/eclipse/emfcloud/coffee/workflow/dsl/Workflow.xtext:14]: Should we update this uri as well?

**Review by ndoschek — COMMENTED** [review]
Thanks Eugen! Building and running the coffee-editor still works as expected! :+1: 

I noticed that the `nsUri` of the following files are still eclipsesource uris. Should we update them as well?
- `Workflow.ecore`
- `workflow-graph.ecore`
- `workflow-notation.ecore`

File: backend/plugins/org.eclipse.emfcloud.coffee.workflow.dsl/src/org/eclipse/emfcloud/coffee/workflow/dsl/Workflow.xtext (hunk 1 above)
Comment (eneufeld) [reply backend/plugins/org.eclipse.emfcloud.coffee.workflow.dsl/src/org/eclipse/emfcloud/coffee/workflow/dsl/Workflow.xtext:14]: I was afraid doing so, as I have no idea what needs to be adapted then. 
Should we do this in a follow up?

Comment (ndoschek) [reply backend/plugins/org.eclipse.emfcloud.coffee.workflow.dsl/src/org/eclipse/emfcloud/coffee/workflow/dsl/Workflow.xtext:14]: That's fine by me!

**Review by ndoschek — APPROVED** [review]

**Pull request merged**